lib/modules/platform/index.spec.ts

```diff
@@ -70,6 +70,7 @@ describe('modules/platform/index', () => {
         {
           hostType: 'bitbucket',
           matchHost: 'api.bitbucket.org',
+          resolvedHost: 'api.bitbucket.org',
```

Comment:
Could you take a look at this change, I am not very sure of it and added this to fix the tests.

Reply:
🤔 looks wrong